compiler/dyno/include/chpl/uast/OpCall.h

```diff
@@ -21,7 +21,7 @@
 #define CHPL_UAST_OPCALL_H
 
 #include "chpl/framework/Location.h"
-#include "chpl/framework/UniqueString.h"
+#include "chpl/framework/global-strings.h"
```

Comment:
I don't think this change is necessary here (since the USTR stuff is in the .cpp, maybe it was here in an  earlier version)